.github/workflows/ci.yml

```diff
@@ -92,16 +102,16 @@ jobs:
           curl -sL -o /usr/local/bin/drush https://github.com/drush-ops/drush-launcher/releases/download/0.6.0/drush.phar && chmod +x /usr/local/bin/drush
           curl -sL -o /usr/local/bin/blt https://github.com/acquia/blt-launcher/releases/download/v1.0.3/blt.phar && chmod +x /usr/local/bin/blt
       ## Create MySQL databases
-      - name: Turn on MySQL and Create DBs
-        run: |
-          sudo /etc/init.d/mysql start
+      #- name: Turn on MySQL and Create DBs
+      #  run: |
+      #    sudo /etc/init.d/mysql start
```

Comment:
Please remove these lines as the service/container takes care of starting mysql.

Reply:
Technically, that code would have started mysql 8. Removed.